code/components/discord/src/DiscordPresence.cpp

```diff
@@ -153,17 +161,81 @@ static void UpdatePresence()
 	}
 }
 
+static void OnRichPresenceStateChange(internal::ConsoleVariableEntry<std::string>* richPresenceState)
+{
+	std::string state = richPresenceState->GetValue();
+
+	if (state != "disabled" && state != "partial" && state != "enabled")
```

Comment:
What do you think if we replace "partial" with "restricted"? It sounds better if we talk about privacy